lib/main/routes/app_routes.dart

```diff
@@ -8,8 +8,8 @@ abstract class AppRoutes {
   static const SESSION = '/session';
   static const COMPOSER = '/composer';
   static const DESTINATION_PICKER = '/destinationPicker';
-  static const MAILBOX_CREATOR = '/mailboxCreator';
+  static const MAILBOX_CREATOR = '$MAILBOX_DASHBOARD/mailboxCreator';
   static const MANAGE_ACCOUNT = '$MAILBOX_DASHBOARD/manage_account';
-  static const IDENTITY_CREATOR = '/identity_creator';
-  static const RULES_FILTER_CREATOR = '/rules_filter_creator';
+  static const IDENTITY_CREATOR = '$MANAGE_ACCOUNT/identity_creator';
+  static const RULES_FILTER_CREATOR = '$MANAGE_ACCOUNT/rules_filter_creator';
```

Comment:
can you say sth about that? What use for?

Reply:
@hoangdat I think he would change the path on the web and makes it clearer.

Reply:
Yep, it should be clear the exact path.